src/pages/components/form/usage.mdx

```diff
@@ -407,6 +423,33 @@ pattern.
 | [Designing for longer forms](/patterns/forms-pattern/#designing-for-longer-forms) | Techniques to help make longer forms less overwhelming, including guidance for accordion and multistep forms.   |
 | [Form variants](/patterns/forms-pattern/#variants)                                | Forms may be presented as dedicated pages, side panels, or dialogs depending on the use case and the situation. |
 
+## AI presence
+
+Form has a modification that takes on the AI visual styling when AI is present
+in the entire container. The AI variants of a form function the same as the
+normal variants except for the addition of the AI slug, which is both a visual
+indicator and the trigger for the explainability popover.
+
+When the entire form is styled for AI, it indicates to the user that AI is
```

Comment:
I think some lead in content could help here, something like: 

> **The AI presence in form can appear in two ways: broadly, over the whole form, or focused, just in certain parts of the form.** When the entire form is styled for AI, it indicates to the user that AI is present in all aspects of the form. If only a limited amount of the form content is generated by AI, then only those components should receive the AI presence styling, not the entire form.

Reply:
That makes sense, thanks @aagonzales!
Should I also add this lead to [Modal](https://carbondesignsystem-fvcy91uyk-carbon-design-system.vercel.app/components/modal/usage/#ai-presence)?